Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md

```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
 Check out the [Users article](../../../Getting-Started/Data/Users/) for an in-depth explanation about Umbraco users in general.
 
 :::note
-Users are environment-specific on Umbraco Cloud. This means that they will not be transferred over when doing a deployment to the next environment - they will need to be added on the different environments on Umbraco Cloud.
+Users are environment-specific on Umbraco Cloud. This means that they will not be transferred over when doing a deployment to the next environment they will need to be added on the different environments on Umbraco Cloud.
+:::
+
+## Invite User through the Umbraco backoffice
+
+As mentioned it is possible to invite new Users to your Umbraco Cloud project through the backoffice as you would on a normal installation of Umbraco.
+
+To invite a User you need to do the following:
+
+1. Go to the backoffice of your Umbraco Cloud project
+2. Go to the User section in the backoffice
+3. Invite new User to the project
+4. Enter name and email and add a User Group to assign access and permissions and optionally enter a new message for the invitation
+
+![Invite User](images/invite_user.png)
+
+Once the User have been invited they will receive an invitation for the project.
+
```

Comment:
Would it make sense to add an H3 (###) heading here, so it's clear that the next part is about how to accept an invite to the backoffice?